tvb/simulator/plot/power_spectra_interactive.py

```diff
@@ -432,19 +431,21 @@ def update_spectra(self):
         self.calc_fft()
         self.plot_spectra()
 
+        # Unused function
```

Comment:
If really unused, then pls remove it. This was part of my requirement: cleanup (aka remove) unused parts.

Reply:
fixed